### DISCUSSION TITLE
Bring back the ability to highlight what you want.

### DISCUSSION THREAD

**Discussion by ICodeDiffusion** [BlafKing/sd-civitai-browser-plus#199] [Ideas]
Before this recent update I was able to highlight the prompts that I wanted, but now when I try it, it instead acts as send to text to image and I loose all my prompts.
Can we have just the button below(that's already there) instead of the whole prompt acting as a button?

**Comment by BlafKing** [comment]
This has been added since the send to text2img button sends all the prompts, the individual buttons can now be used to send partial prompt data to txt2img, a few users requested this feature which is why it was added, I do however understand your concern with the new function.

I'll make it so that it doesn't send anything to txt2img if any text has been highlighted, so if you drag select some text, then it won't send anything to txt2img.
Additionally I'll add a toggle in the SD-WebUI settings that you can use to completely toggle off individual prompt buttons if you don't think it's a feature you'll use! :)

**Comment by BlafKing** [comment]
The mentioned functions are now included in v3.4.0

I'll go and close this request! :)
